handler/asset_handler.go

```diff
@@ -0,0 +1,40 @@
+package handler
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/vkhichar/asset-management/contract"
+	"github.com/vkhichar/asset-management/customerrors"
+	"github.com/vkhichar/asset-management/service"
+)
+
+func ListAssetHandler(asset service.AssetService) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "application/json")
```

Comment:
move set header where you write the response

Reply:
please elaborate